advisories/unreviewed/2024/03/GHSA-rxwq-x6h5-x525/GHSA-rxwq-x6h5-x525.json

```diff
@@ -14,6 +15,25 @@
     }
   ],
   "affected": [
+    {
+      "package": {
+        "ecosystem": "",
```

Comment:
Can the ecosystem be anything other than the supported list? For this CVE, the ecosystems are `deb` and `rpm`